### PR TITLE
RS/YJ/RULE19-29

### PR DESCRIPTION
**Pull request by yunjoonjung-PNNL**
This rule hasn't been tested. I'll test it out when the TCD is available.

### CODE REVIEW

**Review by weilixu — APPROVED** [review]
This is the baseline version of rule 19-28.
Same comments applies to here.
Must address

**Comment by JacksonJ-KC** [comment]
> This rule hasn't been tested. I'll test it out when the TCD is available.

https://github.com/pnnl/ruleset-checking-tool/pull/1090

File: rct229/rulesets/ashrae9012019/section19/section19rule29.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        return {"inapplicable_hvac_sys_list_b": inapplicable_hvac_sys_list_b}
+
+    class HVACRule(RuleDefinitionBase):
+        def __init__(self):
+            super(Section19Rule29.HVACRule, self).__init__(
+                rmrs_used=UserBaselineProposedVals(False, True, False),
+                required_fields={
+                    "$": ["fan_system"],
+                    "fan_system": [
+                        "operation_during_occupied",
```
Comment (JacksonJ-KC) [inline rct229/rulesets/ashrae9012019/section19/section19rule29.py:54]: @yunjoonjung-PNNL same comment as 19-28. Should this instead be `operation_during_unoccupied`?

**Comment by yunjoonjung-PNNL** [comment]
This Rule was tested with TCD and passed as expected.

File: rct229/rulesets/ashrae9012019/section19/section19rule29.py (hunk 1 above)
Comment (yunjoonjung-PNNL) [reply rct229/rulesets/ashrae9012019/section19/section19rule29.py]: Good catch. Addressed.

**Pull request merged**